RankingTests/RankingTests.cs

```diff
@@ -66,7 +106,53 @@ public void TestSaveFile2()
             reader.ReadLine();
             reader.ReadLine();
             int actual = Convert.ToInt32(reader.ReadLine());
-            Assert.AreEqual(15, actual, "Test ranking can save to file");
+            int num = rank.Scores[1].Value;
+            Assert.AreEqual(num, actual, "Test ranking can save to file");
+        }
+        //test: check if ranking obj can sort score objs by value
+        //expected: list of scores is sorted (true)
+        [Test]
+        public void TestSort()
+        {
+            Ranking rank = Setup();
+            rank.Sort();
+            bool actual = true;
+            for (int i = 0; i < rank.Scores.Length; i++)
+            {
+                for (int j = i + 1; j < rank.Scores.Length; i++)
+                {
+                    if (rank.Scores[i].Value > rank.Scores[j].Value) { actual = false; break; }
+                }
+            }
+            Assert.AreEqual(true, actual, "Test ranking can sort scores");
+        }
+        //test: check if ranking obj can remove and sort when there are more than 10 scores
+        //expected: 10 scores after adding and list of scores is sorted (true)
+        [Test]
+        public void TestCheckScoreCount()
```

Comment:
I think this test case should be only for testing the length of the list of scores. We already have a test case for sorting above.

Reply:
Okay